File: PAT (Basic-Level)/C/1063.c
#include <stdio.h>
#include <math.h>

int main(int argc, char const *argv[])
{
    int N;
    scanf("%d", &N);
    int a, b;
    double radius = 0;
    double temp = 0;
    for (int i = 0; i < N; ++i)
    {
        scanf("\n%d %d", &a, &b);
        temp = sqrt(a*a + b*b);
        if(temp > radius){
            radius = temp;
        }
    }
    printf("%.2f", radius);
    return 0;
}

File: PAT (Basic-Level)/Python/1063.py
#!/usr/bin/python3
# -*- coding: utf-8 -*-
import math

N = int(input())
radius = temp = 0
for i in range(N):
    a, b = list(map(int, input().split()))
    temp = math.sqrt(a*a + b*b)
    if temp > radius:
        radius = temp;
print("%.2f" % radius)